.github/workflows/labels.yml

```
@@ -30,7 +30,7 @@ concurrency:
 # Labels will most likely not exist in forks, yet. For this case,
 # we add the issues permission only here.
 permissions:
-  issues: write # needed to create *new* labels
+  issues: write
```

> I'm not sure what you mean. The nixpkgs-ci app needs the issues privilege, otherwise... it can't operate on issues. Which is the point of this PR - marking issues as stale, too. We currently only do PRs via this action.

I saw the old comment that was removed and misinterpreteted it as also applying to this (but just being cleaned up for readability). How about:

```suggestion
  issues: write  # Needed to close issues and add/remove labels
```

---

I think @Pandapip1 was reading the comment as "this is what the permission allows",[^1] however it was actually intended as "this is why we need the permission _here_".


Before, when we only touched PRs needing an `issues` permission required some explanation. Now, it is self-explanatory; no comment required.


[^1]: Unless there's a particularly surprising behaviour, we generally don't describe what something from github's API does; if you want to know that you are better off reading GitHub's docs than our comments :grin: